### ISSUE TITLE
On profiles : Links en/guides/theguidesname = 404

### ISSUE THREAD

**Issue by gregoirelacoste** [dis-moi/extension#1004]
Since the last version of the extension was push in production, the version of the profiles is obsolete on the router and does not follow the extension paths. We must push in production the latest version of the profiles

**Comment by gregoirelacoste** [comment]
@antoinewolff there is an issue with wordpress router could you check please ?

**Comment by antoinewolff** [comment]
@gregoirelacoste : after checking...
- I don't see any redirection nor in the redirection plugin, nor in the .htaccess.
- But... but ! If I flush the Cloudflare cache... it works ! 🎉 : https://www.dismoi.io/en/guides/16/Le-Meme-en-Mieux

**Comment by antoinewolff** [comment]
Yes it works only if you get there from the link in https://www.dismoi.io/en/guides/.

**Comment by antoinewolff** [comment]
It work in french : https://www.dismoi.io/eclaireurs/16/Le-Meme-en-Mieux/

**Comment by MaartenLMEM** [comment]
> 
> 
> > ```
> > * But... but ! If I flush the Cloudflare cache... it works ! tada : https://www.dismoi.io/en/guides/16/Le-Meme-en-Mieux
> > ```
> 
> When I click on your link, I came across a 404 err, you to ?

Same here one Firefox (french language) without any DisMoi extension activated, content is no displayed.
![image](https://user-images.githubusercontent.com/7824485/119729557-16543300-be75-11eb-81ac-fe226b3a491f.png)

**Comment by JalilArfaoui** [comment]
Tell me if you need help @antoinewolff

**Comment by antoinewolff** [comment]
I just checked : now https://www.dismoi.io/en/guides/16/Le-Meme-en-Mieux is redirected to https://www.dismoi.io/en/guides/

**Comment by antoinewolff** [comment]
I just checked : now https://www.dismoi.io/en/guides/16/Le-Meme-en-Mieux is redirected to https://www.dismoi.io/en/guides/

**Comment by gregoirelacoste** [comment]
@antoinewolff indeed, to avoid 404 i redirect to en/guides because it'es impossible to find where to create a fine redirection in WP!

**Comment by gregoirelacoste** [comment]
@lutangar @JalilArfaoui the fix is already in prod. I just did a redirection to /en/guides from wordpress functions.php because it was to hard to find where to do a true redirect

**Comment by gregoirelacoste** [comment]
see : https://github.com/dis-moi/www.dismoi.io/blob/bf537cb543f5ec5b1ecb9ef13c4596929f97694c/themes/divi-child-bulle/functions.php#L678

**Comment by antoinewolff** [comment]
@gregoirelacoste : We've already seen that and @JalilArfaoui is going to make it work better. Thks !

**Comment by gregoirelacoste** [comment]
> @gregoirelacoste : We've already seen that and @JalilArfaoui is going to make it work better. Thks !

I am currently working on it

**Comment by antoinewolff** [comment]
Let us know if it's driving you crazy.

**Comment by gregoirelacoste** [comment]
@JalilArfaoui @antoinewolff if you could check a minute to find a new idea...
